avm/utilities/tools/Set-AVMModule.ps1

```diff
@@ -118,6 +116,7 @@ function Set-AVMModule {
             ################
             if (-not $using:SkipReadMe) {
                 Write-Output "Generating readme for [$resourceTypeIdentifier]"
+                . $using:ReadMeScriptFilePath
```

Comment:
Please resolve conflicts, probably due to readme updates

Reply:
done